### PR TITLE
add python-request versions between 2.6 and 2.18

### LINKED ISSUE #1653: python-requests
<pre>python-requests/2.18.4</pre> is categorized with 2.1 version while it should be 2.18

all the versions are [here](https://pypi.python.org/pypi/requests)

### PR DESCRIPTION
**Pull request by mmorel-35**
fix #1653

### CODE REVIEW

**Comment by mmorel-35** [comment]
I'm gonna add the user-agents added before given by @jaydiablo as test so please wait before validating the pull-request

File: tests/issues/issue-1653.php [hunk 1]
```diff
@@ -0,0 +1,921 @@
...
+            'Device_Brand_Name' => 'unknown',
+            'RenderingEngine_Name' => 'unknown',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'unknown'
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true
+    ],
+    'issue-1653-M' => [
```
Comment (mimmi20) [inline tests/issues/issue-1653.php]: Please remove the duplicate test. This UA is already available on test `issue-1597-NA`

**Comment by mmorel-35** [comment]
Is there any way to run the integrations tests locally or could we summarize all the test failures  instead of first ?

File: resources/user-agents/crawlers/rippers/python-requests.json [hunk 2]
```diff
@@ -1,149 +1,47 @@
...
+          "match": "python-requests/#MAJORVER#.#MINORVER#* CPython/#PLATFORM#",
           "device": "Linux Desktop",
-          "properties": {
-            "Version": "2.2",
-            "MajorVer": "2",
-            "MinorVer": "2"
-          },
           "platforms": [
-            "Linux_64", "Linux_32", "Linux"
+            "Linux_64","Linux_64_amd", "Linux_32", "Linux"
```
Comment (jaydiablo) [inline resources/user-agents/crawlers/rippers/python-requests.json]: Usually you can remove "Linux_32" as the "Linux" platform will catch anything that the "Linux_64" and "Linux_64_amd" platforms don't (and has the same properties defined as "Linux_32".

**Comment by jaydiablo** [comment]
You can run them locally, make sure you `composer install` all dependencies first, then you should be able to run the tests like so:

`./vendor/bin/phpunit -c tests/phpunit-integration.xml.dist ./tests/UserAgentsTest/`

or if you want to do just the Full file tests, for example:

`./vendor/bin/phpunit -c tests/phpunit-integration.xml.dist ./tests/UserAgentsTest/FullTest.php`

**Comment by mmorel-35** [comment]
I have this failure : any idea why ? 

<pre>

2) UserAgentsTest\FullTest::testUserAgents with data set "issue-1597-NJ" ('RSSOwl/2.2.1.201312301314 (Wi...U; en)', array('General RSS', 'RSSOwl', 'Bot/Crawler', '0', 'unknown', 'unknown', '0.0', '0', '0', 'unknown', 'unknown', 'unknown', '0', 'unknown', false, false, false, false, false, false, false, false, false, false, false, false, false, false, false, false, true, true, false, false, false, '0', '0', 'unknown', 'unknown', 'Desktop', 'unknown', 'unknown', 'unknown', 'unknown', 'unknown', 'unknown'), false, true, true)
Expected actual "Device_Type" to be "Desktop" (was "unknown"; used pattern: rssowl/*)
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-'Desktop'
+'unknown'
</pre>

**Comment by jaydiablo** [comment]
Hmm, that issue on `master` is set as "unknown" for the `Device_Type`: https://raw.githubusercontent.com/browscap/browscap/master/tests/issues/issue-1597.php (you'll have to search for "issue-1597-NJ" in there, the file is too big to load withing github's UI).

Perhaps you've modified that test locally?  I don't see that particular test modified in this PR's changes.

**Comment by jaydiablo** [comment]
Thanks @mmorel-35 !!

**Pull request merged**